### PR TITLE
Make logstash node system test more stable

### PR DESCRIPTION
**Pull request by ruflin**
Introduce a 1 sec timeout to help the service to get available and return all fields.

### CODE REVIEW

**Comment by exekias** [comment]
In the past I used grep on the healthcheck, to ensure the missing field is already populated, wdyt?

**Comment by ruflin** [comment]
@exekias Interesting idea. We could definitively do that. Not a fan of regexp foo but probably the most stable solution. Which service did you do that for some copy / paste? Seems like we need in the end a script for the healtcheck :-(

**Comment by exekias** [comment]
```
module/ceph/_meta/Dockerfile:HEALTHCHECK --interval=1s --retries=90 CMD ceph --status | grep HEALTH_OK
module/kibana/_meta/Dockerfile:HEALTHCHECK --interval=1s --retries=120 CMD curl -f http://localhost:5601/api/status | grep '"disconnects"'
module/kubernetes/_meta/Dockerfile:HEALTHCHECK --interval=1s --retries=90 CMD curl -f localhost:10255/stats/summary | grep kube-addon-manager
```

**Comment by ruflin** [comment]
jenkins, retest it

File: metricbeat/module/logstash/node/node_integration_test.go [hunk 1]
```diff
@@ -13,7 +13,6 @@ import (
 )
 
 func TestFetch(t *testing.T) {
-
```
Comment (ruflin) [inline metricbeat/module/logstash/node/node_integration_test.go]: @andrewkroh for you 😆

**Pull request merged**